pkg/specgen/generate/ports_bench_test.go

```diff
@@ -5,11 +5,13 @@ import (
 	"testing"
 
 	"github.com/containers/common/libnetwork/types"
+	"github.com/stretchr/testify/require"
 )
 
 func benchmarkParsePortMapping(b *testing.B, ports []types.PortMapping) {
 	for n := 0; n < b.N; n++ {
-		ParsePortMapping(ports, nil)
+		_, err := ParsePortMapping(ports, nil)
+		require.NoError(b, err)
```

Comment:
This is a benchmark, I am not sure If we want to slow it down with an extra unrelated function call.